### ISSUE TITLE
Allow to set a predefined list of "from" addresses

### ISSUE THREAD

**Issue by gabrielfin** [phpList/phplist3#602]
This is a feature request to (optionally) define a list of "from" addresses, so that the admin user can only choose one of those as a "from line" in a campaign, instead of allowing them to enter whichever address they like.
This would prevent problems, especially with non technical users. If they enter the wrong address, then replies to the campaign would get lost. Or if they enter the wrong domain, there could be deliverability issues or, even worse, being marked as spammers.

**Comment by michield** [comment]
Yes, that's a good idea. @suelaP do you want to create a mantis ticket for this?
